### PR TITLE
[#1031] Export Non English Languages To Prism

### LINKED ISSUE #1031: [Export] Export non-English languages to Prism
Use the language vocabulary https://github.com/inveniosoftware/invenio-rdm-records/blob/master/invenio_rdm_records/fixtures/data/vocabularies/languages.yaml as admissible languages. 

For refresher, the language field in InvenioRDM: https://inveniordm.docs.cern.ch/reference/metadata/#languages-0-n

### PR DESCRIPTION
**Pull request by Meowcenary**
Add support for all languages that appear on production DigitalHub to
export. closes #1031

### CODE REVIEW

File: app/models/invenio_rdm_record_converter.rb [hunk 1]
```diff
@@ -183,7 +183,7 @@ def invenio_metadata
         "subjects": SUBJECT_FIELDS.map{ |subject_field| subjects_for_field(@generic_file.send(subject_field), subject_field) }.compact.flatten.uniq,
         "contributors": contributors(@generic_file.contributor),
         "dates": format_dates(@generic_file.date_created),
-        "languages": @generic_file.language.map{ |lang| lang.present? && lang.downcase == ENGLISH ? {"id": "eng"} : nil }.compact,
+        "languages": @generic_file.language.map{ |lang| LANGUAGES[lang.downcase] ? {"id": LANGUAGES[lang.downcase]} : nil }.compact,
```
Comment (Meowcenary) [inline app/models/invenio_rdm_record_converter.rb:186]: Not using assignment in a conditional :D

File: spec/models/invenio_rdm_record_converter_spec.rb [hunk 2]
```diff
@@ -38,7 +38,7 @@
       date_created: ["2021-1-1"],
       mime_type: 'application/pdf',
       grants_and_funding: ["European Commission 00k4n6c32"],
-      language: ["English"],
+      language: ["Pali", "English", "Fake Language"],
```
Comment (Meowcenary) [inline spec/models/invenio_rdm_record_converter_spec.rb:41]: Simple test case to cover:
- English 
- Non-English and supported
- Not supported

File: app/models/invenio_rdm_record_converter.rb [hunk 3]
```diff
@@ -14,8 +14,8 @@ class InvenioRdmRecordConverter < Sufia::Export::Converter
   ABBR_MONTHNAMES = Date::ABBR_MONTHNAMES.map{ |abbr_monthname| abbr_monthname.downcase if abbr_monthname.present? }
   MONTHNAMES = Date::MONTHNAMES.map{ |monthname| monthname.downcase if monthname.present? }
   SEASONS = ["spring", "summer", "fall", "winter"]
+  LANGUAGES = {"english": "eng", "pali": "pli", "afrikaans": "afr", "polish": "pol", "italian": "ita", "french": "fra"}.with_indifferent_access
   ENG = "eng"
```
Comment (sharpattack) [inline app/models/invenio_rdm_record_converter.rb:18]: Can this be removed?

File: spec/models/invenio_rdm_record_converter_spec.rb (hunk 2 above)
Comment (sharpattack) [reply spec/models/invenio_rdm_record_converter_spec.rb:41]: Is there a value that may not be supported?

**Review by sharpattack — APPROVED** [review]
Looks good to me. Have a couple small questions. Should not block from merging though. We can address non-supported languages in another ticket, if needed.

File: app/models/invenio_rdm_record_converter.rb (hunk 3 above)
Comment (Meowcenary) [reply app/models/invenio_rdm_record_converter.rb:18]: Sure

**Pull request merged**